packages/browser-wallet/src/shared/utils/token-helpers.ts

```diff
@@ -33,6 +33,7 @@ export interface ContractDetails {
 
 export type ContractTokenDetails = TokenIdAndMetadata & {
     balance: bigint;
+    error: string;
```

Comment:
It would be good to document that the empty string is used for no error

Reply:
I think it would be better to make it optional, as that explains the intention. Unless there's a reason for the empty string I'm missing?

Reply:
`error` now optional